terraform/variables.tf

```diff
@@ -20,3 +20,13 @@ variable "billing_account" {
   type        = "string"
   description = "The name of your billing account. Case-sensitive."
 }
+
+variable "project_id" {
+  type        = "string"
+  description = "The id of your project. Case-sensitive."
+}
+
+variable "bucket_name" {
+  type        = "string"
+  description = "The name of your bucket. Case-sensitive."
```

Comment:
this could be improved. What is the bucket for?

Reply:
When you initialize the backend, you need to give it a bucket.

Reply:
It might be helpful to say what the bucket is used for in the description I mean. I assume it's to hold the tfstate?